specification/appcomplianceautomation/resource-manager/Microsoft.AppComplianceAutomation/preview/2022-11-16-preview/examples/Report_CreateOrUpdate.json

```diff
@@ -0,0 +1,117 @@
+{
+  "parameters": {
+    "parameters": {
+      "properties": {
+        "offerGuid": "0000",
+        "timeZone": "GMT Standard Time",
+        "triggerTime": "2022-03-04T05:11:56.197Z",
+        "resources": [
+          {
+            "resourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/myResourceGroup/providers/Microsoft.Network/privateEndpoints/myPrivateEndpoint",
+            "tags": {
+              "key1": "value1"
+            }
```

Comment:
are thee actual tags of the ARM resource above ? Will this operation actually update them ?

Reply:
@raosuhas, Yes, these tags are the ARM resources' tag. But we just store their info, and not update them.